Makefile

```diff
@@ -139,6 +140,10 @@ embed_files: getstatik wrap_registry_chart
 	statik -m -f -src=./assets/embedded-web-files/views -ns webViews -p statikWebViews -dest assets
 	statik -m -f -src=./assets/embedded-web-files/assets -ns webAssets -p statikWebAssets -dest assets
 
+update-app-crd:
+	./scripts/update-app-crd.sh
+# The form above expects the environment variable REV to be set to the desired revision
```

Comment:
what form? maybe `form` -> `script` or `form` -> `target` ?

Reply:
Addressed in rebase, modified original commit